test/integration_test/types.go

```diff
@@ -31,6 +31,10 @@ type TestCase struct {
 	ShouldStartSuccessfully bool
 	// ShouldSkip check env requirements for the test case
 	ShouldSkip func() bool
+	// BackendRequirements contains additional information to construct a px-pure-secret.
+	BackendRequirements *BackendRequirements
```

Comment:
[nit] Do we want to call this `PureBackendRequirements`? Or keep it as-is and if we have more backends in the future we just expand this struct

Reply:
I'll change it to pure backend and we could change it in case we have more.